knowledge-portal/source/source-pipeline-checks.py

```diff
@@ -0,0 +1,119 @@
+import os
+import sys
+from glob import glob
+
+# Search for manifest file
+manifest_file = glob("docs-manifest.txt")[0]
```

Comment:
Given that the manifest has to exist at project root, is a glob here required?

Reply:
It's not required.